### DISCUSSION TITLE
Works latest macOS Monterey 12.2 (21D49)

### DISCUSSION THREAD

**Discussion by chris1111** [chris1111/WirelessAdapterCloverBigSur#9] [Show and tell]
⇩⇩⇩
Works latest macOS Monterey 12.2 (21D49)

![Screen Shot 1](https://user-images.githubusercontent.com/6248794/151589532-eedafb05-b9fe-4bd4-8110-843dc8e6d38a.png)
![Screen Shot 2](https://user-images.githubusercontent.com/6248794/151589540-199b770c-3e7b-41fb-a051-2dfadc5945fc.png)

**Comment by mihaibalaci** [comment]
Any clue if the M1 versions will be supported any time soon?
